src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -699,7 +700,15 @@ class DevTask extends AbstractFeatureTask {
                     // stdout/stderr from the installFeature task is sent to the terminal
                     // only need to log the actual stacktrace when debugging
                     logger.error('Failed to install features from configuration file', e);
-                    if (generateFeatures && !project.configurations.getByName('libertyFeature').dependencies.isEmpty()) {
+
+                    boolean containsConflictMessage = false;
+                    if (e.getCause() != null && e.getCause().getCause() != null && e.getCause().getCause().getCause() != null) {
+                        if (e.getCause().getCause().getCause().getMessage().contains(InstallFeatureUtil.CONFLICT_MESSAGE)) {
+                            // PluginExecutionException from installFeature will be 3 layers deep
+                            containsConflictMessage = true;
+                        }
+                    }
```

Comment:
We only need to check for this conflict message if this check is true:
```
if (generateFeatures && !project.configurations.getByName('libertyFeature').dependencies.isEmpty())
```
Maybe put the conflict check inside of that if statement? Otherwise it will run every time, but it may not be necessary.
If it ends up being messy, and readability is an issue, we could make the conflict check a separate method (containsConflictMessage boolean could be a method call)

Reply:
Good idea, I've updated the logic